lib/logflare/backends/adaptor/postgres_adaptor/pg_repo.ex

```diff
@@ -98,49 +57,22 @@ defmodule Logflare.Backends.Adaptor.PostgresAdaptor.PgRepo do
     |> then(&"log_events_#{&1}")
   end
 
-  @doc """
-  Retunrns a list of migrations to run.
-  """
-  @spec migrations(Source.t()) :: list({pos_integer(), atom()})
-  def migrations(%Source{} = source), do: [{1, AddLogEvents.generate_migration(source)}]
-
-  @doc """
-  Rolls back all migrations
-  """
-  @spec rollback_migrations(Adaptor.source_backend()) :: :ok
-  def rollback_migrations({source, backend}) do
-    repo_module_name = get_repo_module(backend)
-    GenServer.call(via(repo_module_name), {:rollback_migrations_table, {source, backend}})
-  end
-
   @doc """
   Drops the migration table
   """
-  @spec drop_migrations_table(Adaptor.source_backend()) :: :ok
-  def drop_migrations_table({source, backend}) do
-    repo_module_name = get_repo_module(backend)
-    GenServer.call(via(repo_module_name), {:drop_migrations_table, source})
-  end
-
-  @doc """
-  Returns the migrations table name used for a given source
-  """
-  @spec migrations_table_name(Source.t()) :: String.t()
-  def migrations_table_name(%Source{token: token}) do
-    token =
-      token
-      |> Atom.to_string()
-      |> String.replace("-", "_")
-
-    "schema_migrations_#{token}"
+  @spec destroy(Adaptor.source_backend(), timeout()) :: :ok
+  def destroy({source, backend}, timeout \\ 5000) do
```

Comment:
scary word + vague, maybe can be more precise with `destroy_tables`

Reply:
`destroy_instance`?